core/opds_import.py

```diff
@@ -2069,14 +2069,17 @@ def feed_contains_new_data(self, feed):
         last_update_dates = self.importer.extract_last_update_dates(feed)
 
         new_data = False
-        for identifier, remote_updated in last_update_dates:
+        for raw_identifier, remote_updated in last_update_dates:
 
-            identifier = self._parse_identifier(identifier)
+            identifier = self._parse_identifier(raw_identifier)
             if not identifier:
                 # Maybe this is new, maybe not, but we can't associate
                 # the information with an Identifier, so we can't do
                 # anything about it.
-                self.log.info("Ignoring %s because unable to turn into an Identifier.")
+                if raw_identifier:
```

Comment:
We should probably still log something here if we don't have a raw identifier, so there is some evidence in the logs that the identifier is missing.

Reply:
Had the same thought, however this is what the change was in the PR, I can change it to this

Reply:
Yes. I think this would be good to change.